services/iam/functions.go

```diff
@@ -26,6 +26,21 @@ func (c *client) CheckAuthenticationWithContext(ctx context.Context, token, meth
 
 }
 
+func (c *client) CheckAuthenticationByEndpoint(api, token, method string) (user iam_grpcapi.User, err error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
+	defer cancel()
+	return c.CheckAuthenticationByEndpointWithContext(ctx, token, method)
+}
+func (c *client) CheckAuthenticationByEndpointWithContext(ctx context.Context, api, token, method string) (user iam_grpcapi.User, err error) {
```

Comment:
Ingen timeout här, hänger den (alltid) i den ctx som kommer in?

Reply:
Jepp, den sitter i contextet